SwiftUIBestPractices.md

```diff
@@ -0,0 +1,187 @@
+# SwiftUI Best Practices
+
+# File Order
+
+## `Body` placed after init
```

Comment:
```suggestion
### `body` placed after `init`
```

Reply:
I'd also add a sentence after this that explains what the example is like is done for property wrappers being grouped together